packages/manager/apps/web/client/app/hosting/database/copy/hosting-database-copy.html

```diff
@@ -0,0 +1,106 @@
+<div data-ng-controller="HostingDatabaseCopyCtrl as $ctrl">
+    <div
+        data-wizard
+        data-wizard-bread-crumb
+        data-wizard-on-cancel="resetAction"
+        data-wizard-on-finish="copyDatabase"
+        data-wizard-title="'hosting_tab_DATABASES_configuration_copy_title' | translate"
+    >
+        <div
+            data-wizard-step
+            data-wizard-step-on-load="load"
+            data-wizard-step-valid="$ctrl.isStep1Valid()"
```

Comment:
Here you're using `$ctrl` instead of `$scope`. It's not really readable to mix both

Reply:
Validated with David: wizard component does not allow that since it's start functions in its own context.